test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -78,7 +76,7 @@ class CustomPeerCommunicationServiceMock : public PeerCommunicationService {
 class ToriiServiceTest : public testing::Test {
  public:
   virtual void SetUp() {
-    runner = new ServerRunner(std::string(Ip) + ":" + std::to_string(Port));
+    runner = std::make_unique<ServerRunner>(ip + ":0");
```

Comment:
Comment on line 75 seems outdated. Please check other files for similar comments.